deploy/sql/package.json

```diff
@@ -7,10 +7,13 @@
   "author": "Guy Harwood",
   "license": "GPL-3.0",
   "scripts": {
-    "db:up": "node ./sql/migrate.js",
-    "db:data": "node ./sql/seed.js",
-    "new:migration": "node ./sql/create-migration.js",
-    "new:seed": "node ./sql/create-seed.js"
+    "db:up": "node ./schema(WIP)/migrate.js",
```

Comment:
I would prefer not to have special characters - the curly braces - in the directory name.

Reply:
renamed